cmd/new.go

```diff
@@ -104,64 +38,78 @@ func (c *newScriptCmd) run(cmd *cobra.Command, args []string) error { //nolint:r
 	if err != nil {
 		return err
 	}
-
 	if fileExists && !c.overwriteFiles {
-		return fmt.Errorf("%s already exists, please use the `--force` flag if you want overwrite it", target)
+		return fmt.Errorf("%s already exists. Use the `--force` flag to overwrite", target)
 	}
 
 	fd, err := c.gs.FS.Create(target)
 	if err != nil {
 		return err
 	}
 	defer func() {
-		_ = fd.Close() // we may think to check the error and log
+		if cerr := fd.Close(); cerr != nil {
+			if _, err := fmt.Fprintf(c.gs.Stderr, "error closing file: %v\n", cerr); err != nil {
+				panic(fmt.Sprintf("error writing error message to stderr: %v", err))
+			}
+		}
```

Comment:
I'd try to avoid the use of `panic` here, but return whatever error happened (or the result of joining both if more than one happened).

Reply:
Gotcha! I removed the panic :eyes: